cranelift/codegen/src/isa/aarch64/lower/isle.rs

```diff
@@ -286,4 +286,105 @@ where
         let amount = val.value() & u8::try_from(ty.bits() - 1).unwrap();
         ImmShift::maybe_from_u64(u64::from(ty.bits()) - u64::from(amount)).unwrap()
     }
+
+    fn icmp_cond_zero(&mut self, cond: &IntCC) -> Option<IntCC> {
+        match cond {
+            &IntCC::Equal => Some(IntCC::Equal),
```

Comment:
It looks like this isn't changing the matched `IntCC`; could we write this as just `IntCC::Equal | IntCC::SignedGreaterThanOrEqual | ... => Some(cond)`?